vbfml/scripts/preprocess_image.py

```diff
@@ -170,9 +170,10 @@ def plot_rotation(input_file: str, name_save: str, ievent: int, num_events: int)
     process_tag = get_process_tag_from_file(input_file)
 
     # location of the plots -> in a new directory
-    output_dir = pjoin(os.path.dirname(os.path.dirname(input_file)), name_save)
+    outtag = os.path.basename(os.path.dirname(input_file))
+    output_dir = pjoin("output", outtag, name_save)
     if not os.path.exists(output_dir):
```

Comment:
So all the root files are gonna be saved in vbfml/vbfml/scripts/output/... ?

Reply:
Not the root files (I haven't touched those), but the plots will be saved under `scripts/output`. So an example could be this:

```
# Saves plots under scripts/output/jobTag/<saveName>, depending on the saveName parameter
./preprocess_image.py plot-rotation -i ../root/jobTag/tree.root
```

Reply:
Let me know what you think, I think it is cleaner to save the plots under an `output` directory, rather than saving them where the root files themselves are located. Also this makes the output file collection in the pipeline very easy, we just specify the pipeline to collect: `vbfml/scripts/output/**/*pdf` (and png).

Reply:
yes it is way cleaner I agree. It wasn't that efficient when I was working with all the plotted images saved next to the root files
so yes I think it is great like this